### PR TITLE
chapter_11

### PR DESCRIPTION
**Pull request by lrmi0808**
Ready

### CODE REVIEW

**Comment by lrmi0808** [comment]
Ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/account_activations.scss:1]: delete this file please

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (anlvq-1058) [inline app/helpers/account_activations_helper.rb:1]: delete this file please

File: app/views/user_mailer/account_activation.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Sample App</h1>
+
+<p>Hi <%= @user.name %>,</p>
+
+<p>
+Welcome to the Sample App! Click on the link below to activate your account:
```
Comment (anlvq-1058) [inline app/views/user_mailer/account_activation.html.erb:6]: ident

File: test/controllers/account_activations_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/account_activations_controller_test.rb:1]: please delete this file

File: test/mailers/previews/user_mailer_preview.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
```
Comment (anlvq-1058) [inline test/mailers/previews/user_mailer_preview.rb:2]: please delete this file

File: test/mailers/user_mailer_test.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/mailers/user_mailer_test.rb:1]: please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by lrmi0808** [comment]
Ready

**Comment by ngocvt-0484** [comment]
image
please fix it

**Comment by lrmi0808** [comment]
Ready

**Comment by lrmi0808** [comment]
Ready

**Comment by lrmi0808** [comment]
Ready

**Comment by lrmi0808** [comment]
> image
> please fix it

**Comment by lrmi0808** [comment]
Open

**Comment by lrmi0808** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**